easybuild/easyconfigs/l/Libint/Libint-2.0.3-stable-fma.patch

```diff
@@ -0,0 +1,98 @@
+Remove some problematic (and not very useful) FMA header code that does not compile with GCC.
+This is done with a patch, because this problem also affects anything that links against Libint.
+This way, FMA compiler optimization can remain enabled.
```

Comment:
@tovrstra Please also add a line to mention who the author is, something like:

```
author: Toon Verstraelen (UGent)
```

Reply:
Done. Sorry for overlooking this at first instance.